### PR TITLE
1933760-set the parameter value customMessage:true in updateDeploymen…

### PR DESCRIPTION
**Pull request by v-nagarajku**
…tHistory and finally block,

**Task name**: -	AzureRmWebAppDeploymentV4

**Description**: It was observed API calls to kudu utility which caused multiple restarts of the web app (**Web App on Windows**).
We have removed the kudu utility API call to update the deployment status. Now we are updating the deployment status along with the deployment.

**Documentation changes required: ** (Y/N) N

**Added unit tests: ** (Y/N) N

**Attached related issue: ** (Y/N) Y

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

### CODE REVIEW

**Review by JasonFreeberg — APPROVED** [review]

File: Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import path = require('path');
 import { TaskParameters, TaskParametersUtility } from './operations/TaskParameters';
 import { DeploymentFactory } from './deploymentProvider/DeploymentFactory';
 import * as Endpoint from 'azure-pipelines-tasks-azure-arm-rest-v2/azure-arm-endpoint';
+import { Console } from 'console';
```
Comment (PhilipsonJoseph) [inline Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts:6]: Can you remove this if it is not used anywhere

File: Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts [hunk 2]
```diff
@@ -13,23 +14,25 @@ async function main() {
...
+       
     }
     catch(error) {
         tl.debug("Deployment Failed with Error: " + error);
         isDeploymentSuccess = false;
         tl.setResult(tl.TaskResult.Failed, error);
     }
     finally {
-        if(deploymentProvider != null) {
+        if(deploymentProvider != null && isDeploymentSuccess==false) {
```
Comment (PhilipsonJoseph) [inline Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts:32]: Have a space before and after the comparison operator

File: Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts [hunk 3]
```diff
@@ -13,23 +14,25 @@ async function main() {
...
             await deploymentProvider.UpdateDeploymentStatus(isDeploymentSuccess);
         }
-        
+        console.log("isDeploymentSuccess-status: ",isDeploymentSuccess);
```
Comment (PhilipsonJoseph) [inline Tasks/AzureRmWebAppDeploymentV4/azurermwebappdeployment.ts:35]: Can you remove the console log if it is not necessary

File: Tasks/AzureRmWebAppDeploymentV4/task.json [hunk 4]
```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 4,
         "Minor": 198,
```
Comment (PhilipsonJoseph) [inline Tasks/AzureRmWebAppDeploymentV4/task.json:20]: Change the major version to current sprint

**Review by PhilipsonJoseph — CHANGES_REQUESTED** [review]

**Review by PhilipsonJoseph — APPROVED** [review]

**Pull request merged**